task.go

```diff
@@ -330,7 +330,7 @@ func (config TaskConfig) validateInputContainsNames() []string {
 type TaskRunConfig struct {
 	Path string   `json:"path" yaml:"path"`
 	Args []string `json:"args,omitempty" yaml:"args"`
-	Dir  string   `json:"dir",omitempty" yaml:"dir"`
```

Comment:
You can leave it - the separate bug is in our Tracker just so I can do explicit acceptance on it.